torch/tensortypes/nestedtensor/nested.py

```diff
@@ -0,0 +1,215 @@
+import torch
+
+# This implementation is based on NestedTensor 0.0.1
+# RFC: https://github.com/pytorch/pytorch/issues/22169
```

Comment:
This is experimental right? We need to clearly say so in the documentation here.

Reply:
Sure, I can add a sentence